modules/dmrpp_module/build_dmrpp.cc

```diff
@@ -428,6 +428,38 @@ static void get_chunks_for_all_variables(hid_t file, D4Group *group)
         // relies on the varaible's position in the DAP dataset hierarchy.
         D4Attribute *attr = d4_attrs->get("fullnamepath");
         string FQN;
+        // I believe the logic is more clear in this way: 
+        // If fullnamepath exists and the H5Dopen2 fails to open, it should throw an error.
+        // If fullnamepath doesn't exist, we should ignore the error as the reason described below:
+        // (However, we should supress the HDF5 dataset open error message.)  KY 2019-12-02
+        // It's not an error if a DAP variable in a DMR from the hdf5 handler
+        // doesn't exist in the file _if_ there's no 'fullnamepath' because
+        // that variable was synthesized (likely for CF compliance)
+        hid_t dataset = -1;
+        if(attr) {
+            if(attr->num_values() == 1) 
+                FQN = attr->value(0);
+            else
+                FQN = (*v)->FQN();
+            VERBOSE(cerr << "Working on: " << FQN << endl);
```

Comment:
VERBOSE will write to the log file. I think this should be BESDEBUG. Neither of those should include 'cerr' since they write to streams set elsewhere.